tests/poll/test_serializers.py

```diff
@@ -0,0 +1,72 @@
+from bulbs.poll.models      import Poll, Answer
+from bulbs.poll.serializers import PollSerializer, AnswerSerializer
+from bulbs.utils.test       import (
+    BaseIndexableTestCase,
+    make_vcr,
+    mock_vault,
+    random_title,
+)
+
+from .common import SECRETS
+
+vcr = make_vcr(__file__)  # Define vcr file path
+
+class PollSerializerTestCase(BaseIndexableTestCase):
+
+    """Tests for the `PollSerializer`"""
+
+    @vcr.use_cassette()
+    @mock_vault(SECRETS)
+    def test_poll_serialization(self):
+        poll = Poll.objects.create(question_text='good text',
+                title=random_title())
+        serializer = PollSerializer(poll)
+        self.assertEqual(serializer.data['id'], poll.id)
+        self.assertEqual(serializer.data['question_text'], poll.question_text)
+        self.assertEqual(serializer.data['title'], poll.title)
+
+    @vcr.use_cassette()
+    @mock_vault(SECRETS)
+    def test_poll_answer_serialization(self):
+        poll = Poll.objects.create(question_text=u'good text',
+                title=random_title())
+        answer1 = Answer.objects.create(poll=poll, answer_text=u'this is some text')
+        answer2 = Answer.objects.create(poll=poll, answer_text=u'forest path')
+        serializer = PollSerializer(poll)
+        answers_data = serializer.data['answers']
+        self.assertEqual(answers_data, [
+            {
+                'id': 1,
+                'answer_text': answer1.answer_text,
```

Comment:
In general I recommend explicitly putting the expected text string here, to make the expected out put more clear (and also to protect against rare chance where serializer might erroneously mutate object data.